playbooks/all-docker-install.yml

```diff
@@ -17,31 +17,26 @@
     max_hosts: 1
 
   tasks:
-
     # Check '--limit' is used
     - name: Fail if you are targeting more than 1 hosts
       include_tasks: tasks/host-limit-check.yml
       run_once: True
 
-    # Check host OS version
-    - name: Check supported OS and version
-      assert:
-        that:
-          - ansible_distribution|lower == 'debian' and ansible_distribution_version == '10'
-        fail_msg: "{{ ansible_distribution }} {{ ansible_distribution_version }} is not yet supported by this role"
-    
+    - name: Check server OS compatibility
```

Comment:
Convention: `- name: Include checking ...`

Reply:
renamed in https://github.com/SkynetLabs/ansible-playbooks/pull/238/commits/de6d344272248acf2313779c0dbc53758035699c